.github/workflows/release-pr-body.md

```diff
@@ -1,11 +1,9 @@
 At this point, the release of _Timefold Quickstarts_ is ready to be published.
 
-- [x] Release branch has been created.
-- [x] Git tag has been published.
-- [ ] Delete the branch `dry_run`.
-- [ ] Create the version branch (e.g., `1.13.x`) from the `stable` one.
+- Release branch has been created.
+- Git tag has been published.
```

Comment:
Arguably, this can be removed entirely. It doesn't provide any extra information.